specification/regionmove/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/regionmovecollection.json

```diff
@@ -0,0 +1,1807 @@
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/MoveCollectionResultList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/OperationStatus"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "MoveCollections_ListMoveCollectionsBySubscription": {
+            "$ref": "./examples/MoveCollections_ListMoveCollectionsBySubscription.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Migrate/moveCollections": {
+      "get": {
+        "summary": "Get all projects.",
+        "description": "Get all the projects in the resource group.",
+        "operationId": "MoveCollections_ListMoveCollectionsByResourceGroup",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "in": "path",
+            "name": "resourceGroupName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/MoveCollectionResultList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/OperationStatus"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-examples": {
+          "MoveCollections_ListMoveCollectionsByResourceGroup": {
+            "$ref": "./examples/MoveCollections_ListMoveCollectionsByResourceGroup.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ResourceIdentityType": {
+      "description": "The type of identity used for the region move service.",
+      "enum": [
+        "None",
+        "SystemAssigned",
+        "UserAssigned"
+      ],
+      "type": "string"
+    },
+    "Identity": {
+      "description": "Defines the MSI properties of the Move Collection.",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/ResourceIdentityType"
+        },
+        "principalId": {
+          "description": "Gets or sets the principal id.",
+          "type": "string"
+        },
+        "tenantId": {
+          "description": "Gets or sets the tenant id.",
+          "type": "string"
+        }
+      }
+    },
+    "MoveState": {
+      "description": "Defines the MoveResource states.",
+      "enum": [
```

Comment:
Will you consider using x-ms-enum (https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-enum) for better code generation?

Reply:
Sure, i have added this. This is an additional information on the enum and as I have understood does not require any change from service.